pkgs/development/python-modules/tensorpotential/default.nix

```diff
@@ -0,0 +1,28 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildPythonPackage,
+  pythonOlder,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "tensorpotential";
+  version = "0.5.1";
+  setuptools = true;
+
+  src = fetchFromGitHub {
+    owner = "ICAMS";
+    repo = "grace-tensorpotential";
+    rev = "0.5.1";
+    hash = "sha256-nVIHW2aiV79Ul07lqt/juGc8oPYJUeb7TLtqMyOQjGs=";
+  };
+
```

Comment:
You need to define the dependencies from the [pyproject.toml](https://github.com/ICAMS/grace-tensorpotential/blob/4b0314b912ddbcebb54a0812d635b89c9b6223dd/pyproject.toml#L24C1-L35C2)